clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/mark_node_failed.py

```diff
@@ -61,7 +62,17 @@ def make_key(site, node_type, datatore, etcd_key):
 key = make_key(site, node_type, datastore, etcd_key)
 logging.info("Using etcd key %s" % (key))
 
-error_syncer = EtcdSynchronizer(NullPlugin(key), dead_node_ip, etcd_ip=local_ip, force_leave=True)
+if datastore == "cassandra":
+  if (run_command("dpkg-query -W -f='${Status}' clearwater-cassandra | grep -iq installed") == 0):
+    sys.path.append("/usr/share/clearwater/clearwater-cluster-manager/plugins")
+    from cassandra_failed_plugin import CassandraFailedPlugin
```

Comment:
A better way of doing this would be to try importing the plugin and catching `ImportError`